createNearWallet.ts

```
@@ -0,0 +1,30 @@
+    keyStore,
+    nodeUrl: `https://rpc.${networkId}.near.org`,
+    walletUrl: `https://wallet.${networkId}.near.org`,
+    helperUrl: `https://helper.${networkId}.near.org`,
+    headers: {}
+  })
+
+  // 5. Create account instance
+  const account = new Account(near.connection, accountId)
+
+  // ✅ Now you can use `account` to interact with NEAR
+  console.log(`Account loaded: ${accountId}`)
```

Biome (2.3.13)

[error] Don't use console.

The use of console is often reserved for debugging.
Unsafe fix: Remove console.


(lint/suspicious/noConsole)